PowerShell/ScubaGear/baselines/exo.md

```diff
@@ -158,12 +152,9 @@ agency, facilitating phishing attacks. Publishing an SPF policy for each agency
 
 ### Implementation
 
-#### MS.EXO.2.1v1 Instructions
-Identify any approved senders specific to your agency.
-Additionally, see [External DNS records required for SPF](https://learn.microsoft.com/en-us/microsoft-365/enterprise/external-domain-name-system-records?view=o365-worldwide#external-dns-records-required-for-spf) for
-inclusions required for Microsoft to send email on behalf of your domain.
+#### MS.EXO.2.2v2 Instructions
+First, identify any approved senders specific to your agency (see  [Create safe sender lists in EOP](https://learn.microsoft.com/en-us/defender-office-365/create-safe-sender-lists-in-office-365) for tips). SPF allows you to indicate approved senders by IP address or CIDR range. However, note that SPF allows you to [include](https://www.rfc-editor.org/rfc/rfc7208#section-5.2) the IP addresses indicated by a separate SPF policy, referred to by domain name. See [External DNS records required for SPF](https://learn.microsoft.com/en-us/microsoft-365/enterprise/external-domain-name-system-records?view=o365-worldwide#external-dns-records-required-for-spf) for inclusions required for M365 to send email on behalf of your domain.
```

Comment:
I don't think the safe sender lists link is relevant here, as that is a different feature entirely (safe sender list = who can my users receive email from vs SPF = who is approved to send email for _my_ domain). Honestly, we could probably get away without having a link for that first part, and it's going to vary a lot by agency. We could also just say something like, "First, identify any approved senders specific to your agency, e.g., any on-premises mail servers."